src/main/java/uk/ac/ebi/ena/readtools/webin/cli/rawreads/BloomWrapper.java

```diff
@@ -19,59 +22,65 @@
 import java.util.concurrent.atomic.AtomicLong;
 import java.util.stream.Collectors;
 
-import com.google.common.hash.BloomFilter;
-import com.google.common.hash.Funnels;
-
 public class 
 BloomWrapper 
 {
     //Bloom bloom;
-    private final BloomFilter<String> bloom;
-    private final double edup = 0.01;
-    private final AtomicLong adds_no = new AtomicLong();
-    private final AtomicLong susp_no = new AtomicLong();
-    private final int collect_max; 
-    private final Set<String> suspected;
+    private BloomFilter<String> bloom;
+    private double falsePositiveProbability = 0.01;
+    private AtomicLong addCount = new AtomicLong();
+    private AtomicLong possibleDuplicateCount = new AtomicLong();
+    private int possibleDuplicatesRetainLimit;
+    private Set<String> possibleDuplicates;
     
     
     public 
-    BloomWrapper( long expected_reads )
+    BloomWrapper( long expectedReads )
```

Comment:
Possibly this could take the readLimit as parameter and assign the size from this.